### ISSUE TITLE
forecast_day_x

### ISSUE THREAD

**Issue by kuradi** [briis/weatherbit#43]
please add those back :)

**Comment by briis** [comment]
Sure. Can I ask how you use them?

**Comment by kuradi** [comment]
custom:weatherbit-weather-card
otherwise i need to add tons of template sensors....

**Comment by briis** [comment]
No problem, I will add them back. Did not even know there was a custom card for this 😁

**Comment by Landshark77** [comment]
Would love to see these added back.  I use these to dynamically build the WEATHER_LIST object for Tileboard.  Referencing a sensor object in tileboard is much easier than referencing the attributes for each day.

Example code where I build the sensor name dynamically, looping through each day (id):
```
var rain = '&sensor.weatherbit_forecast_day_' + id +'.attributes.precip_prob' + '%';
icon: '&sensor.weatherbit_forecast_day_' + id +'.attributes.icon',
var high = this.parseFieldValue('&sensor.weatherbit_forecast_day_' + id +'.attributes.temperature', item, entity);
var low = this.parseFieldValue('&sensor.weatherbit_forecast_day_' + id +'.attributes.templow', item, entity);
```
Result:
![Screenshot from 2021-12-22 08-41-29](https://user-images.githubusercontent.com/27645488/147109828-c7e22b00-4f32-4ab3-b425-91b3ea8709ad.png)

**Comment by briis** [comment]
I have already done that, and will be in release 1.0.1. However I did change the naming of some of the attributes to be in line with Home Assistant standards, so you might need to do minor edits in your code above. Example: `precip_prop` is now `precipitation_probability`

**Comment by briis** [comment]
Released 1.0.1 where this is included.
